front-end/src/index.tsx

```diff
@@ -30,15 +30,20 @@ const httpLink = new HttpLink({
 });
 
 const tokenRefreshLink = new TokenRefreshLink({
-	accessTokenField: 'data.token',
+	accessTokenField: 'token',
 	fetchAccessToken: getRefreshedToken,
 	handleError: (err:any) => {
-		console.warn('Your refresh token is invalid. Try to login again');
-		console.error(err);
+		console.error('There has been a problem with your fetch operation: ', err);
+	},
+	handleFetch: (accessToken) => {console.log('access token',accessToken) ;storeLocalStorageToken(accessToken)},
```

Comment:
write in multiple lines

Reply:
OOps that shouldn't be there actually, stray log.